### DISCUSSION TITLE
Some Converters are subclassing ValueConverterExtension, what is the purpose of that ?

### DISCUSSION THREAD

**Discussion by Cfun1** [xamarin/XamarinCommunityToolkit#1035] [Q&A] [answered]
I see that some converters are sub classing `ValueConverterExtension` and some others not.
https://github.com/xamarin/XamarinCommunityToolkit/blob/caa3ebc71335d68e4a69758def542b322ccdbf90/src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/BoolToObjectConverter.shared.cs#L18

But I am not sure what is the purpose of that, is it to be able to use it as a xaml markup extension? If yes I don't see how? also in the sample app converters are only consumed as a resource and not as a xaml markup extension.

Tried for BoolToObjectConverter with something like :
`Color="{converters:BoolToObjectConverter IsToggled, FalseObject=#0000FF, TrueObject=#FF0000}"`

>     No constructor for type 'BoolToObjectConverter' has 1 parameters.

Is something needs to be improved ?

**Comment by AndreiMisiukevich** [comment, marked as answer]
```IsVisible="{Binding Items, Converter={xct:ListIsNotNullOrEmptyConverter}}"```

**Reply by Cfun1** [reply]
Oh this is magic :) was looking at the code but couldn't figure out how it works, thank you :) this is also the answer to https://github.com/xamarin/XamarinCommunityToolkit/issues/1002
